job.go

```diff
@@ -163,37 +166,23 @@ func (jl JobList) Sort(sorter func(i, j int) bool) JobList {
 }
 
 //DoesJobContainTaskRange evaluates whether the XML marshalled tasks value contains the regex indicating a sequence of tasks.
-func DoesJobContainTaskRange(j Job) (bool, error) {
-	regex, err := regexp.Compile(TASKRANGEIDENTIFIERREGEX)
-
-	if err != nil {
-		return false, err
-	}
-
-	return regex.MatchString(j.Tasks.Source), nil
+func DoesJobContainTaskRange(j Job) bool {
+	return TaskRangeRegex.MatchString(j.Tasks.Source)
```

Comment:
hmm I don't like how this is connected as a higher scoped variable specific to the function this is getting invoked in. I'd either elevate the taskrangeregex to a parameter or make that truly global, or just get rid of the function all together and use the MatchString directly (though I don't mind the increased clarity in what it's doing

Reply:
I just moved it to a more global definition. It didn't feel like it was buying anything more than complexity to pass it in as a parameter to the function, and if it was already  global it would be easier to work and its definition / population only occurs once.